tests/console/yast2_lan.pm

```diff
@@ -78,8 +78,12 @@ sub run {
     type_string $domain;
     assert_screen 'test-yast2_lan-1';
 
+    my $timeout = 90;
+    if (check_var('ARCH', 'aarch64')) {
+        set_var("timeout", "180");
+    }
```

Comment:
simply use `=` to give value to a perl variable.
if you wanna double time that could be like this `$timeout *= 2 if check_var('ARCH', 'aarch64');`